metrics/report/report_dockerfile/tidy_scaling_nc.R

```diff
@@ -96,12 +96,15 @@ if (length(latencydata[[1]]) <= 20 && perc_count == 5) {
 			upper_name = names(ltpt)[perc_count-n+2]
 			upper_prev_name = names(ltpt)[perc_count-n+1]
 			perc_dist = abs(n-perc_mid)
-			alpha = 0.4 * (n / perc_mid)**2
+			alpha = 0.7 * ((n+1) / (perc_mid+1))**2
 			latp = latp + geom_ribbon(data=latencydata[latencydata$testname==plot_test,],aes_string(x="n_pods",ymin=lower_name,ymax=lower_next_name,fill="testname"),alpha=alpha)
 			latp = latp + geom_ribbon(data=latencydata[latencydata$testname==plot_test,],aes_string(x="n_pods",ymin=upper_prev_name,ymax=upper_name,fill="testname"),alpha=alpha)
 		}
-		# Draw median line
-		latp = latp + geom_line(data=latencydata[latencydata$testname==plot_test,],aes_string(x="n_pods",y=names(ltpt)[perc_mid+1],color="testname"))
+		median_index = match("p50", names(ltpt))
+		if (!is.na(median_index)) {
+			# Draw median line
+			latp = latp + geom_line(data=latencydata[latencydata$testname==plot_test,],aes_string(x="n_pods",y=names(ltpt)[median_index],color="testname"))
```

Comment:
@askervin - I was wondering if there might be some built in R functionality for generating a similar chart with 'fading percentile bands' from the raw data - looks like it is very similar to a fanchart/fanplot - not sure if we can utilise that with our data under R though (did not dig too far): https://journal.r-project.org/archive/2015-1/abel.pdf